CommonLibSF/include/RE/B/BGSMod.h

```diff
@@ -31,7 +31,22 @@ namespace RE::BGSMod
 			{
 			public:
 				// members
```

Comment:
Move this down to the members

Reply:
Could you please rephrase? I don't understand.

Reply:
Move the comment to be right above the class members at the bottom